inc/cavalcade_runner_to_cloudwatch/namespace.php

```diff
@@ -102,25 +103,17 @@ function on_end_job( Worker $worker, Job $job, string $status ) {
 	$error_output_property->setAccessible( true );
 	$error_output = $error_output_property->getValue( $worker );
 
-	send_event_to_stream(
```

Comment:
Is there a reason we were using this copied function instead of the `send_events_to_stream` defined in `inc/cloudwatch_logs`?